php/php-zstd/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           php 1.1
+
+github.setup        kjdev php-ext-zstd 0.7.4
+fetch.type          git
+
+name                php-zstd
+categories-append   devel
+platforms           darwin
+maintainers         nomaintainer
+license             MIT
+
+php.branches        7.0 7.1 7.2 7.3
+
+description         Zstandard compression
+
+long_description    This extension allows Zstandard compression from PHP
+
+checksums           rmd160  7f4db240b31dcd23e80d7f9a727f9c60a270ab03 \
+                    sha256  4755cb11e7189980877cbed0003c7b919a429e48aecc503980fad12b09190c23 \
+                    size    15306
+
+post-fetch {
+    system -W ${worksrcpath} "git submodule update --init"
+}
```

Comment:
This should go inside the if block below so it isn't run for the stub port.

Reply:
Amended, thanks!